File: template.js
const { readFileSync } = require('fs');
const { argv, stdout } = require('process');
const { range, zip, countNoDup, occurs, array, print, dict, norm, direct4, direct8, ascIntCmp, descIntCmp, ord, printTable, isOrder } = require('../util.js')
const { count, sum, avg, median, min, max, gcd, lcm, Stack } = require('/usr/local/lib/node_modules/mathball')
const PriorityQueue = require('/usr/local/lib/node_modules/js-priority-queue')

const set = new Set();
const pq = new PriorityQueue({ initialValues: [], comparator: ascIntCmp });
let lines = readFileSync(argv[2]).toString().split(/\r?\n/).slice(0, -1);
let n = lines.length;
let m = lines[0]?.length;
let list = [];
let visit = array([n], 0);
let table = array([n, m], 0);
let ansi = 0;
let anss = "";

lines.forEach((line, idx) => {
    let parts = line.split(",").flat().map(x => parseInt(x));

});

console.log(ansi);
console.log(anss);

File: template.py
import sys

sys.path.append("..")
from collections import defaultdict, Counter, deque
import numpy as np
from queue import PriorityQueue
from functools import cmp_to_key
from itertools import cycle
from util import direct4, direct8

uniq = set()
pq = PriorityQueue()
dictl, dicti = defaultdict(list), defaultdict(int)
ansi, anss = 0, ""

f = open(sys.argv[1], "r")
lines = [line.strip() for line in f.readlines()]
n, m = len(lines), max(len(line) for line in lines)
array, table = [], np.zeros((n, m))

for i, line in enumerate(lines):
    parts = [int(x) for x in line.split(",")]

print(ansi)
print(anss)
